OpenRA.Mods.Common/Widgets/SimpleLabelWidget.cs

```diff
@@ -0,0 +1,22 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA.Mods.Common.Widgets
+{
+	public class SimpleLabelWidget : LabelWithTooltipWidget
```

Comment:
There is used Label in .yaml for MAP_TITLE, which does not have tooltip. I can't use LabelWithTooltipWidget because its ctor accepts world. Do you have an idea how to implement it better?
SimpleLabelWidget allows me to reuse all code with minimal change and custom lobby screen work like so far.